### DISCUSSION TITLE
Is there a method to display unix timestamps with various formats?

### DISCUSSION THREAD

**Discussion by red8888** [Textualize/rich#2052] [Ideas]
I didn't see anything, but want to check this isn't already a feature.

It would be cool to provide a standard unix timestamp to rich and make it easy to print it in various formats.

If rich is sort of like the view layer I think it makes sense for it to be able to convert unix timestamps. Is this already possible?

**Comment by sanders41** [comment]
Maybe I am missing what you are saying but you can do the conversion without Rich and then Rich can print it.

```py
from datetime import datetime

from rich import print

timestamp = 1647027405
print(datetime.fromtimestamp(timestamp))
```

**Comment by willmcgugan** [comment]
Unix timestamps are integers, so Rich can't know they are meant to be rendered as times. Perhaps you can can give an example of what formats you want to display, and what kind of API?
